pubsub/message.go

```diff
@@ -90,6 +91,32 @@ func toMessage(resp *pb.ReceivedMessage, receiveTime time.Time, doneFunc iterDon
 	return msg, nil
 }
 
+// AckResult holds the result from a call to Ack or Nack.
+//
+// Call Get to obtain the result of the Ack/NackWithResult call. Example:
+//   // Get blocks until Ack/NackWithResult completes or ctx is done.
+//   ackStatus, err := r.Get(ctx)
+//   if err != nil {
+//       // TODO: Handle error.
+//   }
+type AckResult = ipubsub.AckResult
+
+// AcknowledgeStatus represents the status of an Ack or Nack request.
+type AcknowledgeStatus = ipubsub.AcknowledgeStatus
+
+const (
+	// AcknowledgeStatusSuccess indicates the request was a success.
```

Comment:
Not necessary to repeat the name of the enum in the comment.

Reply:
This is the Go linter style. Running `golint ./...` gives the following error otherwise:

`pubsub/message.go:108:2: comment on exported const AcknowledgeStatusSuccess should be of the form "AcknowledgeStatusSuccess ..."`